CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # bedrock-authn-token ChangeLog
 
+## 5.0.0 - 2022-xx-xx
+
+### Changed
+- **BREAKING**: Update length of `nonce` in `generateNonce` to 6 instead of 9.
```

Comment:
```suggestion
- **BREAKING**: Update length of `nonce` in `generateNonce` to 6 instead of 9.
  Additional care must be taken in deployments to prevent too many attempts to
  input (or "guess") a nonce at any verification endpoints in a short period of time.
```